resources/linux/debian/control.in

```diff
@@ -1,6 +1,6 @@
 Package: <%= appFileName %>
 Version: <%= version %>
-Depends: git, gconf2, gconf-service, libgtk-3-0 (>= 3.9.10), libudev0 | libudev1, libgcrypt11 | libgcrypt20, libnotify4, libxtst6, libnss3 (>= 2:3.22), python, gvfs-bin, xdg-utils, libcap2, libx11-xcb1, libxss1, libasound2 (>= 1.0.16), libxkbfile1
+Depends: git, gconf2, gconf-service, libgtk-3-0 (>= 3.9.10), libudev0 | libudev1, libgcrypt11 | libgcrypt20, libnotify4, libxtst6, libnss3 (>= 2:3.22), python, gvfs-bin, xdg-utils, libcap2, libx11-xcb1, libxss1, libasound2 (>= 1.0.16), libxkbfile1, libcurl3, libcurl-openssl1.0-dev
```

Comment:
I found out that you can do `libcurl3 | libcurl4` here to prevent users on Ubuntu 18.x distros from having to do installation ping-pong, but https://github.com/desktop/dugite-native/issues/109 is my current fresh hell related to this split 🤦‍♂️

Reply:
Oh, nice!  I think that's probably a good solution to try here too.